client/my-sites/domains/domain-management/security/index.jsx

```diff
@@ -0,0 +1,191 @@
+		let icon, text;
+
+		switch ( sslStatus ) {
+			case SSL_PROVISIONING:
+				text = translate( 'Provisioning' );
+				break;
+			case SSL_ACTIVE:
+				icon = 'check_circle';
+				text = translate( 'Enabled' );
+				break;
+			case SSL_DISABLED:
+				icon = 'info';
+				text = translate( 'Disabled' );
+				break;
+		}
+
+		const statusClassNames = classNames( 'security__status', sslStatus );
+
+		return (
+			<span className={ statusClassNames }>
+				{ icon && <MaterialIcon icon={ icon } /> }
+				{ text }
+			</span>
+		);
+	}
+
+	getStatusDescription( domain ) {
+		const { selectedSite, purchase, translate } = this.props;
+		const sslStatus = this.getSSLStatus( domain );
+
+		if ( SSL_PROVISIONING === sslStatus ) {
+			return (
+				<p>
+					{ translate(
+						'Due to some changes to your domain, we need to generate a new SSL certificate to activate your HTTPS encryption. This process should only take a couple hours at most so if you’re running into delays please let us know so we can help you out.'
+					) }
+				</p>
+			);
+		}
+
+		if ( SSL_DISABLED === sslStatus ) {
+			return (
+				<React.Fragment>
+					<p>
+						{ translate(
+							'We have disabled HTTPS encryption because your domain has expired and is no longer active. Renew your domain to reactivate it and turn on HTTPS encryption.'
+						) }
+					</p>
+					{ selectedSite.ID && ! purchase && <QuerySitePurchases siteId={ selectedSite.ID } /> }
+					<RenewButton
+						primary={ true }
+						purchase={ purchase }
+						selectedSite={ selectedSite }
+						subscriptionId={ parseInt( domain.subscriptionId, 10 ) }
+						redemptionProduct={ domain.isRedeemable ? this.props.redemptionProduct : null }
+						reactivate={ ! domain.isRenewable && domain.isRedeemable }
+						tracksProps={ { source: 'security-status', domain_status: 'expired' } }
+					/>
+				</React.Fragment>
+			);
+		}
+
+		return (
+			<React.Fragment>
+				<p>
+					{ translate(
+						'Strong encryption is critical to ensure the privacy and security of your site. This is what you get with HTTPS encryption on WordPress.com:'
+					) }
+				</p>
+				<ul>
+					{ [
+						translate( 'Trust indicators that reassure your visitors your site is safe ' ),
+						translate( 'Secure data transmission of data sent through forms' ),
+						translate( 'Safe shopping experience with secure payments' ),
+						translate( 'Protection against hackers trying to mimic your site' ),
+						translate( 'Improved Google search rankings' ),
+						translate( '301 redirects for all HTTP requests to HTTPS' ),
+					].map( ( feature, index ) => (
+						<li key={ index }>{ feature }</li>
+					) ) }
+				</ul>
+			</React.Fragment>
+		);
+	}
+
+	getContent() {
+		const { domain, translate } = this.props;
+
+		if ( ! domain ) {
+			return null;
+		}
+
+		return (
+			<React.Fragment>
+				<CompactCard className="security__header">
+					<span>{ translate( 'HTTPS encryption' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'HTTPS encryption: on' )` *ES Score: 9*